tests/ui/unwrap.rs

```diff
@@ -10,7 +10,23 @@ fn unwrap_result() {
     let _ = res.unwrap();
 }
 
+// Should not get linted.
+#[test]
+fn with_test() {
```

Comment:
This doesn't get linted, but not because the linting code prevents this. Code marked with `#[test]` attributes in an UI test file will get removed, before the test is run.